workflow/util/util.go

```diff
@@ -813,6 +847,7 @@ func FormulateRetryWorkflow(ctx context.Context, wf *wfv1.Workflow, restartSucce
 	deletedNodes := make(map[string]bool)
 	deletedPods := make(map[string]bool)
 	var podsToDelete []string
+	var resettedParentGroupNodes []string
```

Comment:
sorry to be annoying but can this be changed to "resetParentGroupNodes" since the past tense verb is "reset"?

Reply:
(same for all uses of this word)

Reply:
Definitely! I’ll update this. Thanks!